packages/@guardian/source-react-components/src/choice-card/ChoiceCardGroup.tsx

```diff
@@ -68,6 +74,9 @@ export const ChoiceCardGroup = ({
 	...props
 }: ChoiceCardGroupProps): EmotionJSX.Element => {
 	const groupId = id ?? generateSourceId();
+	const applyContainerTopMargin = () => {
```

Comment:
this reads to me like it have a side affect. also, does it need to be a function?

how do you feel about this?

```js
const topMargin = (label && !hideLabel) || supporting || error ? containerTopMargin : '';
```

Reply:
Good shouts! Thanks 🙏